forge/licensing/index.js

```diff
@@ -40,8 +50,18 @@ module.exports = fp(async function (app, opts, next) {
         },
         active: () => activeLicense !== null,
         get: (key) => {
-            return key ? activeLicense && activeLicense[key] : activeLicense
-        }
+            if (!key) {
+                return activeLicense
+            }
+            if (activeLicense) {
+                if (Object.hasOwn(activeLicense, key)) {
+                    return activeLicense[key]
+                }
+                return undefined
+            }
+            return defaultLimits[key]
```

Comment:
`licenseApi.get(key)` seems to return default values i no `activeLicense` however if a key is not provided, the empty `activeLicense` is returned 

Should this be return `activeLicense || defaultLimits` ?

<br>

Also, the defaults see a little high (150/50/50/50) - if a user doesnt apply a licence they get large defaults? 
I assume this is acceptable since without a license the product is CE (not EE)?

Reply:
> Also, the defaults see a little high (150/50/50/50) - if a user doesnt apply a licence they get large defaults?
I assume this is acceptable since without a license, the product is CE (not EE)?

Those are the agreed defaults for the unlicensed, ie CE, platform.

Reply:
IF the user doesn't have a license applied and it asks for a license value, it returns the default.

If the user does have a license applied, it returns whatever the license specifies (defaulting to `0` if the license doesn't say anything).